src/renderer/components/notification-comp.tsx

```diff
@@ -318,15 +361,6 @@ export default class NotificationComp extends React.Component<{}, IState> {
     }
   }
 
-  /**
-   * Clears a active notification flash interval
-   */
-  private clearFlashInterval(): void {
```

Comment:
So we had support for flashing notifs already?

Reply:
well spotted.. I had exactly the same comment when I started digging into it but I didn't find anything related to flashing in C2 code base and I don't think we have custom notifications in 1.5